bcgov/mds
# Feature/mds 4000 project summary

# Main

- Create Project Summaries tab on the Mines dashboard
- Created AddEditProjectSummary form component to handle CRUD operations
- Created ProjectSummaryFileUpload component to handle document upload
- Created new private route component ProjectSummaryPage

# Other

- Extended DocumentTable to support multiple entities rather than just Variances

# How to test

- Login to Minespace
- Access any mine from the My Mines list
- Click on the Project Summaries tab(far right)
- Click "Create a new Project Summary"
- Fill in details and submit(page will reload to edit screen after submission)
- Update any fields/documents and save
- Navigate back to Project Summaries tab and ensure the new records are represented in the list

# Notes

- N/A
